cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -191,6 +193,7 @@ public MasterServiceMain() {
     this.logAppenderInitializer = injector.getInstance(LogAppenderInitializer.class);
     this.zkClient = injector.getInstance(ZKClientService.class);
     this.leaderElection = createLeaderElection();
+    this.fileContext = injector.getInstance(FileContext.class);
```

Comment:
I see. Can you open a JIRA in Twill for that as well. We should have that in Location so that we can revert hacks like this.